components/playlist/playlist_download_request_manager.cc

```diff
@@ -71,6 +72,9 @@ void PlaylistDownloadRequestManager::CreateWebContents() {
     // |web_contents_| is created on demand.
     content::WebContents::CreateParams create_params(context_, nullptr);
     web_contents_ = content::WebContents::Create(create_params);
+    auto web_prefs = web_contents_->GetOrCreateWebPreferences();
+    web_prefs.allow_cosmetic_filtering = true;
+    web_contents_->SetWebPreferences(web_prefs);
```

Comment:
Oo, it's a common anitpattern.
Another client can update WebPreferences and drop you changes.

If I'm not mistaken the preferred way to use [BraveContentBrowserClient](https://github.com/brave/brave-core/blob/stop-checking-google-translate/browser/brave_content_browser_client.h#L35)

Here the method that should be overrided:
https://github.com/chromium/chromium/blob/main/chrome/browser/chrome_content_browser_client.h#L386-L387